tachyon/math/polynomials/multivariate/linear_combination_unittest.cc

```diff
@@ -0,0 +1,68 @@
+#include "tachyon/math/polynomials/multivariate/linear_combination.h"
+
+#include "gtest/gtest.h"
+
+#include "tachyon/base/containers/container_util.h"
+#include "tachyon/math/finite_fields/test/finite_field_test.h"
+#include "tachyon/math/finite_fields/test/gf7.h"
+#include "tachyon/math/polynomials/multivariate/multilinear_extension.h"
+
+namespace tachyon::math {
+namespace {
+
+const size_t kMaxDegree = 4;
+
+using F = GF7;
+using Poly = MultilinearExtension<MultilinearDenseEvaluations<F, kMaxDegree>>;
+using Point = Poly::Point;
+using PolyPtr = std::shared_ptr<Poly>;
+
+class LinearCombinationTest : public FiniteFieldTest<F> {};
+
+}  // namespace
+
+// Tests |AddTerm()| and |SumOfProducts()|
+TEST_F(LinearCombinationTest, AddTermAndSumOfProducts) {
+  LinearCombination<Poly> linear_combination(kMaxDegree);
+  F coefficient1 = F::Random();
```

Comment:
I think `coefficient_1` would be better for consistency(<-> `coefficient_2`).
```suggestion
  F coefficient_1 = F::Random();
```